### ISSUE TITLE
issue

### ISSUE THREAD

**Issue by laoma023012** [spyder-ide/spyder#6786]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "/home/laoma/anaconda3/lib/python3.6/site-packages/spyder/app/mainwindow.py", line 2132, in closeEvent
    if self.closing(True):
  File "/home/laoma/anaconda3/lib/python3.6/site-packages/spyder/app/mainwindow.py", line 2186, in closing
    self.save_current_window_settings(prefix)
  File "/home/laoma/anaconda3/lib/python3.6/site-packages/spyder/app/mainwindow.py", line 1415, in save_current_window_settings
    self.maximize_dockwidget(restore=True)# Restore non-maximized layout
  File "/home/laoma/anaconda3/lib/python3.6/site-packages/spyder/app/mainwindow.py", line 2268, in maximize_dockwidget
    self.last_plugin.dockwidget.setWidget(self.last_plugin)
AttributeError: 'NoneType' object has no attribute 'dockwidget'
Error in atexit._run_exitfuncs:

## Version and main components

* Spyder Version: 3.2.4 
* Python Version: 3.6.3
* Qt Versions: 5.6.2, PyQt5 5.6 on Linux

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.13.3 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
rope >=0.9.4     :  0.10.5 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
psutil >=0.3     :  5.4.0 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.26.1 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.1.0 (OK)
pylint >=0.25    :  1.7.4 (OK)

```

**Comment by ccordoba12** [comment]
This is fixed in our latest release (3.2.8). Please update spyder by opening a terminal (xterm, konsole, etc) and running

    conda update spyder

**Comment by CAM-Gerlach** [comment]
For reference, duplicate of as #6169 , and fixed in Spyder 3.2.6, released several months ago.
